.github/workflows/dotnetcore.yml

```diff
@@ -41,14 +44,24 @@ jobs:
           path: '**/*.trx'
           reporter: dotnet-trx
 
-    # PR opened, push prelease Carter package to feedz
+    - if: >-
+        (github.event_name == 'pull_request')
+        ||
+        (github.event_name == 'push' && startsWith(github.ref, 'refs/tags') && !startsWith(github.ref, 'refs/tags/newtonsoft-'))
+      name: Create Carter template NuGet package
+      run: >-
+        dotnet pack ./template/Template.csproj
+        --configuration Release
+        --nologo
+
+    # PR opened, push prelease Carter and CarterTemplate packages to feedz
     - if: github.event_name == 'pull_request'
-      name: Push prerelease Carter package to Feedz
```

Comment:
Not sure we'd push to Feedz on a pull reqeust

Reply:
As mentioned above, it's your call.

Reply:
As mentioned above, your decision, we can adapt the logic.

Reply:
With #305, we stopped publishing packages when a PR is open, only when a PR is merged or the repo is tagged.